### PR TITLE
BLAC-672: Adjust Collection Level heading

### PR DESCRIPTION
**Pull request by CEdwardsBlasikiewicz**

### CODE REVIEW

File: app/components/arclight/document_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  <div class='al-show-breadcrumb'>
+    <nav aria-label="breadcrumb">
+      <%= render breadcrumb_component.new(presenter: presenter) %>
+    </nav>
+  </div>
+</div>
+
+<div class="title-container">
+  <%= content_tag :h1 do %>
+    <%= "Guide to the " + document.normalized_title %>
```
Comment (yetti) [inline app/components/arclight/document_component.html.erb:11]: Add this to the localisation file for easier re-use and if we ever end up wanting to display other languages.

Please update the text in the `ShowPresenter` when you do.

File: app/presenters/show_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ShowPresenter < Arclight::ShowPresenter
+  def heading
+    "Guide to the " + document.normalized_title
```
Comment (yetti) [inline app/presenters/show_presenter.rb:5]: Will this clobber the environment tag at the beginning of the tab title in the lower envs?

**Review by yetti — CHANGES_REQUESTED** [review]
Please add feature specs for the change in the title iheading as well as the tab title.

File: app/presenters/show_presenter.rb (hunk 2 above)
Comment (CEdwardsBlasikiewicz) [reply app/presenters/show_presenter.rb:5]: Nope, see attached image.
image

**Review by yetti — CHANGES_REQUESTED** [review]
I still don't see any specs for the title change. Can you please write a feature spec for this.

**Review by yetti — APPROVED** [review]
Thanks for adding the tests and fixing the Github workflow. Looks great!

**Pull request merged**